server/plugins/org.eclipse.glsp.ide.editor/src/org/eclipse/glsp/ide/editor/internal/actions/EnableDefaultToolsAction.java

```diff
@@ -0,0 +1,26 @@
+/********************************************************************************
+ * Copyright (c) 2021 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+package org.eclipse.glsp.ide.editor.internal.actions;
+
+import org.eclipse.glsp.server.actions.Action;
+
+public class EnableDefaultToolsAction extends Action {
+   public static final String ID = "enable-default-tools";
```

Comment:
Offtopic: The id of the `EnableDefaultToolsAction` should probably use the same syntax that we use for other action ids.
i.e. `enableDefaultTools` instead of  `enable-default-tools`

Reply:
Agreed but the IDs for those actions come from Sprotty, so there is not much we can do at here.